app/app.py

```diff
@@ -242,18 +248,26 @@ def _get_predictions(
     return data["generated_text"]
 
 
-def query_llm(prediction_endpoint: str, messages: Dict[str, str]) -> str:
+def query_llm(
+    prediction_endpoint: str,
+    messages: Dict[str, str],
+    temperature: float,
+    max_length: int,
+) -> str:
     """Query endpoint to fetch the summary.
 
     Args:
         prediction_endpoint (str): Prediction endpoint.
         messages (Dict[str, str]): Dict of message containing prompt and context.
+        temperature (float): inference temperature
+        max_length (int): max response length in tokens
 
     Returns:
         str: Summarised text.
     """
     with st.spinner("Loading response..."):
-        payload = _create_payload(messages)
+        payload = _create_payload(messages, temperature, max_length)
+        print(payload)
```

Comment:
do we want to log this instead using `logging.info`?

Reply:
Yep, fair